css/style.css

```
@@ -0,0 +1,220 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 100%;
```

Use knowledge from Jonas's course
```suggestion
  font-size: 62.5%;
```